eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Utxow.hs

```diff
@@ -598,8 +551,7 @@ instance
   where
   inject = UtxoFailure
 
--- | Calculate the set of hash keys of the required witnesses for update
--- proposals.
+-- | Deprecated.
```

Comment:
Any reason we don't deprecate it with {- DEPRECATED #-}  ? 
I see it's only called from the two now deprecated functions, `propWits` and `witsVKeyNeededGov`

Reply:
There is no point. It is not an exported function. DEPRECATED pragma is for letting downstream users know that a function will disappear soon and they should act accordingly. Once deprecated functions in this module that use this `proposedUpdatesWitnesses` are removed then this function will result in an unused function warning and can be removed as well. So, it is enough to leave a comment that this is truly an unused function.

Reply:
Aha, makes sense.